### PR TITLE
Addelectronics

### PR DESCRIPTION
**Pull request by NEEHITGOYAL**
- **Rushal Shah**
- **The channel is all about electronics and communication tutorials **:
- **i have made a new section of electronics and communication as there was no section made for this field**:
- **this youtuber teaches the concept of electronics to its core not like do cram this or that. It really helped me a lot and also developed my interest in electronics field ***:

Issue:#42

### CODE REVIEW

**Comment by NEEHITGOYAL** [comment]
@JoseDeFreitas kindly review and merge pr

File: readme.md [hunk 1]
```diff
@@ -1001,3 +1001,12 @@ Featured playlists: `Getting started with your Raspberry Pi`. \
 <br/>
 
 [🔼 Back to the top](#contents)
+
+## Electronics and Communication
+
+All tutorials and videos of every single old as well as new electronic component available in market . how to get started with electronics stuff ,every useful resource and also get advice/recommendation which product to use and why from persons having a much more experience in this field
```
Comment (JoseDeFreitas) [inline readme.md:1007]:
```suggestion
All tutorials and videos of every single old as well as new electronic components available in the market. How to get started with electronics stuff, every useful resource, and also get advice/recommendation which product to use and why from persons having much more experience in this field.
```
Fix some little text errors.

File: readme.md [hunk 2]
```diff
@@ -1001,3 +1001,12 @@ Featured playlists: `Getting started with your Raspberry Pi`. \
...
+
+[<img align="left" height="94px" width="94px" alt="All about electronics channel's avatar" src="https://yt3.ggpht.com/a/AATXAJy96DtZPu2EJwh5eXuSn6-ru4wz1C4xFTs8GBcwYA=s176-c-k-c0x00ffffff-no-rj"/>](https://www.youtube.com/c/ALLABOUTELECTRONICS)
+
+[**All About Electronics**](https://www.youtube.com/c/ALLABOUTELECTRONICS) \
+Content about: Electronics and Communication \
```
Comment (JoseDeFreitas) [inline readme.md:1012]: Below this line, the channel entry should have a "Featured playlists" field. Please read the [guidelines](https://github.com/JoseDeFreitas/awesome-youtubers/blob/main/contributing.md) so you can see how to.

**Review by JoseDeFreitas — CHANGES_REQUESTED** [review]
Hello @NEEHITGOYAL! Please see the change requests I commented on above.
Apart from that, the channel is good. The tutorials are straight to the point. Thanks for requesting this channel.

File: readme.md (hunk 1 above)
Comment (NEEHITGOYAL) [reply readme.md]: done @JoseDeFreitas

File: readme.md [hunk 3]
```diff
@@ -1001,3 +1001,13 @@ Featured playlists: `Getting started with your Raspberry Pi`. \
...
+
+## Electronics and Communication
+
+All tutorials and videos of every single old as well as new electronic components available in the market. How to get started with electronics stuff, every useful resource, and also get advice/recommendation which product to use and why from persons having much more experience in this field.
+
+[<img align="left" height="94px" width="94px" alt="All about electronics channel's avatar" src="https://yt3.ggpht.com/a/AATXAJy96DtZPu2EJwh5eXuSn6-ru4wz1C4xFTs8GBcwYA=s176-c-k-c0x00ffffff-no-rj"/>](https://www.youtube.com/c/ALLABOUTELECTRONICS)
+
+[**All About Electronics**](https://www.youtube.com/c/ALLABOUTELECTRONICS) \
+Content about: Electronics and Communication \
+Featured playlists: `BJT`, `Analog Electronics`, `Network Analysis`. \
```
Comment (JoseDeFreitas) [inline readme.md:1013]:
```suggestion
Featured playlists: `BJT`, `Analog Electronics`, `Network Analysis`. \
<br/>
```

**Review by JoseDeFreitas — APPROVED** [review]
Thank you for fixing those issues @NEEHITGOYAL!
I consider the channel is awesome 😄. Thank you!

**Pull request merged**